forward_engineering/config.json

```diff
@@ -1,7 +1,7 @@
 {
 	"type": "ddl",
 	"ddlType": "plugin",
-	"mode": "yugabyte-ysql",
+	"mode": "pgsql",
```

Comment:
why?

Reply:
`mode` is responsible for script highlighting, ace-editor doesn't have styles for `yugabyte-ysql`